Libraries/StyleSheet/StyleSheetTypes.js

```diff
@@ -12,7 +12,12 @@
 
 const AnimatedNode = require('../Animated/src/nodes/AnimatedNode');
 
-export type ColorValue = null | string;
+import type {ColorValue, ProcessedColorValue} from './ColorValueTypes';
+import {PlatformColor} from './ColorValueTypes';
+
+export type {ColorValue, ProcessedColorValue};
```

Comment:
Why does StyleSheetTypes need to re-export ProcessedColorValue?

I imagine the purpose of re-exporting ColorValue is to be backwards compatible.

Reply:
I just switched it back so that ColorValue is just defined and exported here.